### PR TITLE
[Livre] Add block patterns

### PR DESCRIPTION
**Pull request by kjellr**
This PR adds an initial set of block patterns for Livre. Original mockups are [in this Figma file](https://www.figma.com/file/ALdcQWdMu3wmdoNv59VwYl/Livre?node-id=2%3A7).

## Headers

Screen Shot 2022-01-07 at 10 11 30 AM

Screen Shot 2022-01-07 at 10 11 41 AM

## Footers

Screen Shot 2022-01-07 at 10 13 23 AM

## Pages

Screen Shot 2022-01-07 at 10 14 11 AM

### CODE REVIEW

File: livre/parts/header.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
...
-<div class="wp-block-group alignwide" style="padding-top:var(--wp--custom--spacing--small, 1.25rem);padding-bottom:var(--wp--custom--spacing--large, 10rem)"><!-- wp:navigation {"layout":{"type":"flex","setCascadingProperties":true,"justifyContent":"left"},"overlayMenu":"always","overlayBackgroundColor":"background","overlayTextColor":"foreground","style":{"typography":{"fontStyle":"normal","fontWeight":"300"}},"fontSize":"x-large"} -->
+<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"bottom":"var(--wp--custom--spacing--large, 10rem)","top":"var(--wp--custom--spacing--small, 1.25rem)"}}},"layout":{"type":"flex","justifyContent":"left"}} -->
+<div class="wp-block-group alignfull" style="padding-top:var(--wp--custom--spacing--small, 1.25rem);padding-bottom:var(--wp--custom--spacing--large, 10rem)"><!-- wp:navigation {"layout":{"type":"flex","setCascadingProperties":true,"justifyContent":"left"},"overlayMenu":"always","overlayBackgroundColor":"background","overlayTextColor":"foreground","style":{"typography":{"fontStyle":"normal","fontWeight":"300"}},"fontSize":"x-large"} -->
```
Comment (kjellr) [inline livre/parts/header.html:2]: This was always supposed to be `alignfull`, not sure why/how it got to be alignwide here. 🤷

File: livre/inc/patterns/footer-nav-left.php [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Footer with left navigation block pattern
+ */
+return array(
+	'title'      => __( 'Footer with left navigation', 'livre' ),
```
Comment (pbking) [inline livre/inc/patterns/footer-nav-left.php:6]: I believe this one should be labeled "Footer with right navigation".

File: livre/inc/patterns/header-title-tagline-social.php [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+	'content'    => '<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"bottom":"var(--wp--custom--spacing--large, 10rem)","top":"var(--wp--custom--spacing--small, 1.25rem)"}}},"layout":{"type":"flex","justifyContent":"space-between"}} -->
+					<div class="wp-block-group alignfull" style="padding-top:var(--wp--custom--spacing--small, 1.25rem);padding-bottom:var(--wp--custom--spacing--large, 10rem)"><!-- wp:navigation {"layout":{"type":"flex","setCascadingProperties":true,"justifyContent":"left"},"overlayMenu":"always","overlayBackgroundColor":"background","overlayTextColor":"foreground","style":{"typography":{"fontStyle":"normal","fontWeight":"300"}},"fontSize":"x-large"} -->
+
+					<!-- wp:page-list {"isNavigationChild":true,"showSubmenuIcon":true,"openSubmenusOnClick":false} /-->
+					<!-- /wp:navigation -->
+
+					<!-- wp:group -->
+					<div class="wp-block-group"><!-- wp:site-title {"textAlign":"center","style":{"typography":{"textTransform":"uppercase"}}} /-->
+
+					<!-- wp:site-tagline {"textAlign":"center","fontSize":"x-small"} /--></div>
```
Comment (pbking) [inline livre/inc/patterns/header-title-tagline-social.php:18]: There is a difference between design and implementation of one of the headers in that there's a fair bit more space between site title and tagline which I think could be addressed by [adjusting](https://github.com/Automattic/themes/pull/5282/files#diff-943023e9d7a2c7192ec3cfc9268caf6399db6ba25ed804c986b2501a19e5bee9R18) the tagline's margin:

Design:
image

View:
image


```
<!-- wp:site-tagline {"textAlign":"center","style":{"spacing":{"margin":{"top":"2px"}}},"fontSize":"x-small"} /-->
```

File: livre/inc/patterns/footer-nav-left.php (hunk 2 above)
Comment (kjellr) [reply livre/inc/patterns/footer-nav-left.php:6]: 😄 Whoops, good catch. Should be sorted out in 4788105.

File: livre/inc/patterns/page-about.php [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+					<!-- wp:column {"width":"66.66%"} -->
+					<div class="wp-block-column" style="flex-basis:66.66%"><!-- wp:image {"id":59,"sizeSlug":"full","linkDestination":"none"} -->
+					<figure class="wp-block-image size-full"><img src="' . esc_url( get_template_directory_uri() ) . '/assets/images/writing.jpg" alt="' . esc_attr__( 'Vintage photograph of a woman writing a letter.', 'livre' ) . '"/></figure>
+					<!-- /wp:image --></div>
+					<!-- /wp:column --></div>
+					<!-- /wp:columns -->
+
+					<!-- wp:spacer {"height":"16px"} -->
+					<div style="height:16px" aria-hidden="true" class="wp-block-spacer"></div>
+					<!-- /wp:spacer -->
```
Comment (pbking) [inline livre/inc/patterns/page-about.php:57]: This (and a few other) spacers seem to be giving 'invalid block' errors.

image

Looking at the markup I'm not sure why exactly, but when Gutenberg corrects it to "100px" tall.

**Review by mikachan — APPROVED** [review]
These all look good to me 👍

**Review by pbking — APPROVED** [review]
These LGTM

**Pull request merged**